### ISSUE TITLE
Failed to parse source map from ...

### ISSUE THREAD

**Issue by ariwoot** [boringdesigners/boring-avatars#43]
Hello and thanks for your work!
After having installed your library I get this warning:

`WARNING in ./node_modules/boring-avatars/build/index.js
[1] Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
[1] Failed to parse source map from '/Users/User/Work/list/project/client/node_modules/boring-avatars/build/index.js.map' file: Error: ENOENT: no such file or directory, open '/Users/User/Work/list/project/client/node_modules/boring-avatars/build/index.js.map'`

The library works even with that warning, still I wondered if there's a workaround to fix this.
Thank you again :)

**Comment by josepmartins** [comment]
Hey @ariwoot, can't reproduce it. Have you tried to delete `node_modules` folder and re-run `npm install` (or yarn)?

**Comment by Ailurus13** [comment]
Hello,

Having the same problem here on a fresh create-react-app with only boring-avatars installed.

`WARNING in ./node_modules/boring-avatars/build/index.js    
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map from 'C:\Users\trist\Documents\prog\boringavatar-test\node_modules\boring-avatars\build\index.js.map' file: Error: ENOENT: no such file or directory, 
open 'C:\Users\trist\Documents\prog\boringavatar-test\node_modules\boring-avatars\build\index.js.map'`

**Comment by ashleyberry** [comment]
@josepmartins have tried deleting `node_modules` folder and re-running `npm install` to no avail - the error still persists.

**Comment by michalzubkowicz** [comment]
Same here.

**Comment by jansphil** [comment]
I'm experience the same issue on a react project with plain js.

This commit https://github.com/boringdesigners/boring-avatars/commit/834303e27405b109bfcd8a6681886c8db3c1ff8c does remove the file after the build process. Using an pretty old version https://github.com/boringdesigners/boring-avatars/tree/v1.4.2 works for me.

**Comment by josepmartins** [comment]
Seem the last Create React App 5.0 throws this type of warnings by source-map-loader and there's a PR opened to fix it https://github.com/facebook/create-react-app/pull/11752, other packages are getting the same warnings https://github.com/facebook/create-react-app/discussions/11767.

Anyways, I've just included the sourcemap in the build again 👍 and release and new `1.6.2` patch version. Tested with a new React Create App and worked without any warnings.

Thanks everyone 👋
